docs/UserGuide.md

```diff
@@ -47,7 +47,8 @@ The application will then prompt the user to enter the following fields:
 - `SUMMARY`: can be in a natural language format
 - `DETAIL`: can be in a natural language format
 
-There can be 0 or multiple `DETAIL` added for an Event Flashcard.
+There can be 0 or multiple `DETAIL` added for an Event Flashcard. To stop entering details, just enter an empty line
+by hitting `RETURN`.
```

Comment:
`ENTER`/`RETURN`? It's a small issue but on Windows the key is called Enter :)

Reply:
Oops I'll update haha